scripts/rollup/results.json

```diff
@@ -1,8 +1,8 @@
 {
   "bundleSizes": {
     "react.development.js (UMD_DEV)": {
-      "size": 65962,
-      "gzip": 16741
+      "size": 65927,
+      "gzip": 16735
```

Comment:
dash-it-all, didn't mean to update this in this commit. Also not sure why it changed again.

Reply:
-rc.1 is a different number of chars than -beta.5 :)